typescript/packages/subsurface-viewer/src/storybook/SubsurfaceViewer.stories.jsx

```diff
@@ -326,6 +324,34 @@ const meshMapLayerPng = {
     colorMapName: "Physics",
 };
 
+const meshMapLayerBig = {
```

Comment:
Where is this used?

Reply:
It is used int the story?! The point is to have a big map which takes some tine to load

Reply:
It's not used in any of those stories. You moved it to the .tsx file.

Reply:
Your're right my mistake :-)